app/controllers/caseload_controller.rb

```diff
@@ -84,13 +83,21 @@ def filter_allocations(allocations)
   end
 
   def load_pom
-    @pom = PrisonOffenderManagerService.get_signed_in_pom_details(
-      current_user,
-      active_prison_id
-    )
+    @pom = StaffMember.new(params[:staff_id])
+  end
+
+  def ensure_signed_in_pom_is_this_pom
+    user = Nomis::Elite2::UserApi.user_details(current_user)
+    unless staff_id == user.staff_id || current_user_is_spo?
```

Comment:
think you forgot to check that we are actually a POM?

Reply:
@starswan  Oh that is a good point. I think I had assumed that that the signed in staff member could only be a pom or an spo.